src/stubs.ts

```diff
@@ -87,6 +87,11 @@ const isMountedComponent = (
   props: ({ [key: string]: unknown } & VNodeProps) | null | undefined
 ) => isComponent(type) && props && props['ref'] === MOUNT_COMPONENT_REF
 
+const isAsyncComponentWrapper = (type: VNodeTypes) =>
+  isComponent(type) &&
+  type.__asyncLoader &&
+  type.name === 'AsyncComponentWrapper'
```

Comment:
Maybe add a comment here indicating that `AsyncComponentWrapper` is the name given by `vue-next` to `defineAsyncComponent`.